Tests/MetaTests/Visitors/CopyVisitorTests.swift

```diff
@@ -0,0 +1,16 @@
+import XCTest
+import Meta
+import SnapshotTesting
+import SwiftSyntax
+
+class CopyVisitorTests: XCTestCase {
+    func testGeneratedCopyMethod() throws {
+        let input = URL(fileURLWithPath: String(#file))
```

Comment:
**suggestion** we can use the tricky extension on `Bundle` to get the local path

```swift
class DummyClass { }
extension Bundle {
    static var local: Bundle { Bundle(for: DummyClass.self) }
}
```

And you can write:
```
Bundle.local.resourceURL.appendingPathComponent....
```